### PR TITLE
[Evaluation] Add accuracy evaluation module for LLM

### PR DESCRIPTION
**Pull request by abenmao**

### CODE REVIEW

**Comment by abenmao** [comment]
@pujiang2018 Done formatting~

**Comment by abenmao** [comment]
@pujiang2018 removed all "import pdb" and modified the copy of logits.

**Review by pujiang2018 — APPROVED** [review]
